blocks/hero/hero.css

```diff
@@ -3,6 +3,14 @@
   padding: 0;
 }
 
+.section.hero-container[data-backgroundcolor="section-bg-color-white"] {
+  background-color: var(--c-white);
+}
+
+.section.hero-container[data-backgroundcolor="section-bg-color-white"] .hero-right-container .hero-image-container::before {
+  background: linear-gradient(90deg, #FFF 2.33%, #fffffff2 24.08%, #ffffffb3 44.28%, #d9d9d900 77.69%);
```

Comment:
please use the small case for color code

Reply:
done @Vaitheesaran-nagalingam